src/build.d

```diff
@@ -260,9 +260,11 @@ alias autoTesterBuild = makeRule!((builder, rule) {
 });
 
 /// Returns: the rule that builds the lexer object file
-alias lexer = makeRule!((builder, rule) => builder
+alias lexer = makeRuleWithArgs!((MethodInitializer!BuildRule builder, BuildRule rule,
+                                 string suffix, string[] extraFlags)
+    => builder
     .name("lexer")
-    .target(env["G"].buildPath("lexer").objName)
+    .target(env["G"].buildPath("lexer" ~ suffix).objName)
     .sources(sources.lexer)
     .deps([versionFile, sysconfDirFile])
     .msg("(DC) LEXER_OBJ")
```

Comment:
Different `lexer` targets should display different messages. Ditto for `backend`,
```suggestion
    .msg("(DC) LEXER" ~ suffix)
```